### PR TITLE
update contribution section, typos

### PR DESCRIPTION
**Pull request by ErichDylus**

### CODE REVIEW

**Comment by KenCarvAPI** [comment]
Thanks Erich.
I guess overall you think communicating any upfront terms is a bad idea. I wasn't 100% sure myself to be honest.
If you have any other ideas to approach this lmk.

**Comment by ErichDylus** [comment]
Well it was more the "experience" qualification. I think either a "bounty" (up front deliverable explanation and grant amount) or open-ended guidelines without a subjective qualification make the most sense

**Comment by KenCarvAPI** [comment]
good point

File: docs/explore/introduction/contributing.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Head to the community (bounty board)[notion.io].
 ### Submit your own proposal
 
 We welcome proposals that bring new ideas or sub-teams to the DAO. Generally
-engaging the DAO through our community in a constructive manor will help
+engaging the DAO through our community in a constructive manner will help
 sentiment check a proposal and help shape it to align to current contributors.
 If you are proposing an initiative that would require collaboration with a
 current sub team, we suggest co-ordinating proactively and outlining how you
```
Comment (dcroote) [inline docs/explore/introduction/contributing.md:75]: "co-ordinating" --> "coordinating"

**Review by dcroote — APPROVED** [review]
Nice! Found one additional typo

**Pull request merged**